dashboard/views.py

```diff
@@ -11,7 +11,12 @@ def dashboard(request):
     }
     if not request.session.get("is_login", None):
         return redirect("landingpage:index")
-    return render(request, "dashboard/index.html", context)
+
+    user_type = request.session.get("user_type", None)
+    if user_type == UserType.ADMIN_STAFF:
+        return redirect("dashboard:admissions:index")
+    else:
+        return render(request, "dashboard/index.html", context)
```

Comment:
This statement might break the flow, we should be showing an invalid credentials pages (exists within the high school templates) instead of redirecting the user to dashboard (there'll be nothing on the dashboard).

Reply:
I haven't updated the else condition here, because you should be redirecting to the student's dashboard page. So ideally, once this code is in, you need to update it to show the student dashboard in the else condition. That's what should happen here.

Reply:
You can add a todo here for now but I believe we shouldn't redirect to the student's dashboard since the user was directed to admission's staff in the first place.

Reply:
No, it was not redirecting to the admin dashboard after login. If you check the login/views.py, you can see that it is being redirected to just "dashboard:dashboard", after the sessions was implemented. That's why this logic is required. But I'll add the TODO now, so that it's easier for you.